src/Connections/HttpConnection.cs

```diff
@@ -9,9 +9,11 @@
 using System.Security.Cryptography;
 using System.Net;
 using System.IO;
+using System.Collections.Specialized;
 
 using WebSocketSharp;
 using WebSocketSharp.Server;
+using _7DTDWebsockets.Extensions;
```

Comment:
Remove `System.Threading.Tasks`, `System.Collections.Specialized`, and `_7DTDWebsockets.Extensions` imports